crates/router/src/core/admin.rs

```diff
@@ -94,18 +94,22 @@ pub async fn get_merchant_account(
     let merchant_details = merchant_account
         .merchant_details
         .parse_value("MerchantDetails")
-        .change_context(errors::ApiErrorResponse::InternalServerError)?;
+        .change_context(errors::ApiErrorResponse::InternalServerError)
+        .attach_printable("Unable to parse value, MerchantDetails")?;
```

Comment:
these printable's are redundant, already these message are added in parse_value, so here and other places like this not required.